tests/sch/transform-attribute.test.tsx

```diff
@@ -18,9 +18,11 @@ test("svg should have data-real-to-screen-transform attribute", () => {
 
   // Extract the transform value and verify it's a valid matrix string
   const match = svg.match(/data-real-to-screen-transform="([^"]+)"/)
-
-  // @ts-ignore
-  expect(match[1]).toMatchInlineSnapshot(
-    `"matrix(54.5454545455,0,0,-54.5454545455,600,300)"`,
-  )
+  expect(match).not.toBeNull() // Ensure match exists before accessing [1]
+  if (match) {
```

Comment:
this is not the correct fix here, you're basically saying "sometimes don't test this"

Reply:
i would revert and do `bun test --update-snapshots test/path` so that i can see how it changed

Reply:
I've fixed it with upgrades, bun